tensorflow/python/ops/sparse_ops_test.py

```diff
@@ -30,6 +30,7 @@
 from tensorflow.python.ops import array_grad  # pylint: disable=unused-import
 from tensorflow.python.ops import gradient_checker_v2 as gradient_checker
 from tensorflow.python.ops import math_ops
+from tensorflow.python.ops import array_ops
```

Comment:
This should be moved below after the line importing `array_grad` (as the imports have to be sorted)

Reply:
Done, thanks!